### PR TITLE
Update maintainer name

### PR DESCRIPTION
**Pull request by ameliabradley**
Signed-off-by: Amelia Bradley <bradley@bitwise.io>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @agunde406 @cleckhardt @davececchi @dcmiddle @dnewh @dplumb94 @jsmitchell @leebradley @peterschwarz @rberg2 @rbuysse @RyanLassigBanks @shannynalayna @vaporos
+*       @agunde406 @cleckhardt @davececchi @dcmiddle @dnewh @dplumb94 @jsmitchell @ameliabradley @peterschwarz @rberg2 @rbuysse @RyanLassigBanks @shannynalayna @vaporos
```
Comment (shannynalayna) [inline CODEOWNERS:1]: Re: the other organization comment, your name would go first in this file, I believe

File: MAINTAINERS.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
 | Name | GitHub | RocketChat |
 | --- | --- | --- |
 | Andi Gunderson | agunde406 | agunde |
```
Comment (shannynalayna) [inline MAINTAINERS.md:6]: I think your name would be organized above Andi's, as it is organized by first name and M is before N in the alphabet.

File: CODEOWNERS (hunk 1 above)
Comment (ameliabradley) [reply CODEOWNERS]: Updated!

File: MAINTAINERS.md (hunk 2 above)
Comment (ameliabradley) [reply MAINTAINERS.md]: Cool, updated!

**Pull request merged**